packages/govern-console/src/services/finances.ts

```diff
@@ -0,0 +1,85 @@
+async function getTokenPrice(tokenSymbol: string) {
+  const apiUrl = `https://api.0x.org/swap/v1/price?sellToken=${tokenSymbol}&buyToken=USDC&sellAmount=1000000000000000000`;
+  try {
+    const rawResponse = await fetch(apiUrl, {
+      method: 'GET',
+      headers: {
+        Accept: 'application/json',
+      },
+    });
+
+    const jsonResponse = await rawResponse.json();
+
+    if (rawResponse.ok) {
+      return {
+        price: jsonResponse.price,
+        error: null,
+      };
+    }
+
+    return {
+      price: null,
+      error: { code: jsonResponse.code, message: jsonResponse.message },
+    };
+  } catch (err) {
+    console.log('ERROR-Fetching price', err);
+  }
+}
+
+/**
```

Comment:
Pls add comment here that this is related to KPI options migration, and only 2 DAOs did it. link to https://blog.aragon.org/uma-kpi-options-airdrop-now-live-for-aragon-govern-daos/